packages/webpack/src/migrations/update-15-6-3/webpack-config-setup.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  formatFiles,
+  logger,
+  readProjectConfiguration,
+  Tree,
+  updateProjectConfiguration,
+} from '@nrwl/devkit';
+import { forEachExecutorOptions } from '@nrwl/workspace/src/utilities/executor-options-utils';
+import { WebpackExecutorOptions } from '../../executors/webpack/schema';
+
+export default async function (tree: Tree) {
+  forEachExecutorOptions<WebpackExecutorOptions>(
+    tree,
+    '@nrwl/webpack:webpack',
+    (
+      options: WebpackExecutorOptions,
+      projectName,
+      targetName,
+      _configurationName
+    ) => {
+      if (options?.['isolatedConfig']) {
```

Comment:
Nitpick: This can be just `options?.isolatedConfig`?

Reply:
Once your `isolatedConfig` PR is merged, it can! :P